resources/views/site/files.blade.php

```diff
@@ -188,8 +47,8 @@
                     </tr>
                 </thead>
                 <tbody>
-                    @if($sop->count() > 0)
-                        @foreach($sop as $f)
+                    @if($$fileCategory->count() > 0)
```

Comment:
Are the two dollar signs intentional? This doesn't appear to cause any issues.

Reply:
This is 100% required, as we are referring to a variable variable

Reply:
Got it. https://www.php.net/manual/en/language.variables.variable.php

Reply:
@kjporter Coming back here - is there a way to do this without the variable variable? I don't like this - it's pretty confusing IMO